jetstream/cli.py

```diff
@@ -217,11 +214,21 @@ def validate_config(path):
         click.echo(f"Validate {file}", err=False)
 
         spec = AnalysisSpec.from_dict(toml.load(file))
-        spec.resolve(dummy_experiment)
+        config = spec.resolve(dummy_experiment)
 
         # check if there is an experiment with a matching slug in Experimenter
         slug = os.path.splitext(os.path.basename(file))[0]
         if collection.with_slug(slug).experiments == []:
             click.echo(f"No experiment with slug {slug} in Experimenter.", err=False)
+        else:
+            # dry run experiment analysis with the config file
+            # this will make sure config file contents are valid
+            ctx.invoke(
+                run,
+                date=config.experiment.start_date,
```

Comment:
If we stick with the dry-run model and not the synthetic query model, I think we should do this for all days between the start_date and end_date.

Reply:
Calling `rerun` will handle some of this. At least for dates in the past...